examples/coverage/module.yaml

```diff
@@ -0,0 +1,20 @@
+# Have feedback? Share it here https://surveys.jetbrains.com/s3/Amper-feedback-form
```

Comment:
Could you please add a test that ensures the coverage works to `e2e-test/.../ExamplesTest`?

Reply:
Added a test to ExamplesTest. Should I also create a testData/projects project to test setting the onCheck for each report?